### PR TITLE
Improve bindist makefile

### PR DESCRIPTION
**Pull request by hasufell**
Pipeline started: https://gitlab.haskell.org/maerwald/haskell-language-server/-/pipelines/48410

<a href="https://gitpod.io/#https://github.com/haskell/haskell-language-server/pull/2746"></a>

### CODE REVIEW

**Review by pepeiborra — APPROVED** [review]

**Comment by hasufell** [comment]
Don't merge, freebsd is broken

**Comment by hasufell** [comment]
Ehm

**Comment by pepeiborra** [comment]
Next time, remove the "merge me" label as well. I have given you maintainer rights in case you tried and couldn't

**Pull request merged**
